sources/api/early-boot-config/src/provider/aws.rs

```diff
@@ -15,7 +15,8 @@ use crate::provider::local_file::{local_file_user_data, USER_DATA_FILE};
 pub(crate) struct AwsDataProvider;
 
 impl AwsDataProvider {
-    const IDENTITY_DOCUMENT_FILE: &'static str = "/etc/early-boot-config/identity-document";
+    const IDENTITY_DOCUMENT_FILE: &str = "/etc/early-boot-config/identity-document";
```

Comment:
Actually the only redundant lifetime was on `USER_DATA_FILE`, but I was asked to remove it from `IDENTITY_DOCUMENT_FILE` for consistency. Which rust toolchain are you currently using? Do you get the same error for `USER_DATA_FILE` or just `IDENTITY_DOCUMENT_FILE`?

Reply:
I did get it for both constants.

```bash
$ cargo --version
cargo 1.63.0
```

Reply:
I saw this running `cargo make check` on a local change that I had rebased on to the latest from `develop`. Although I got the failure locally, I did push the change up to my PR, and the GH Actions that ran against the change all passed. Those show they were running `1.64.0`.

I will try updating to 1.64.0 as well and see if I get better results. Will report back here if that's the case so we can point anyone else that runs in to this here.

Reply:
It looks like rust ≤ **1.63** wants `'static` on `IDENTITY_DOCUMENT_FILE` and  the newly added `FALLBACK_REGION`, with `USER_DATA_FILE` remaining `&str`. Since the Bottlerocket SDK uses **1.64** this isn't a super big problem, but I'll post a fix so we can increase our toolchain compatibility.